devices/philips.js

```diff
@@ -1739,7 +1739,8 @@ module.exports = [
         fromZigbee: [fz.ignore_command_on, fz.ignore_command_off, fz.ignore_command_step, fz.ignore_command_stop,
             fz.legacy.hue_dimmer_switch, fz.battery],
         exposes: [e.battery(), e.action(['on-press', 'on-hold', 'on-hold-release', 'up-press', 'up-hold', 'up-hold-release',
-            'down-press', 'down-hold', 'down-hold-release', 'off-press', 'off-hold', 'off-hold-release'])],
+            'down-press', 'down-hold', 'down-hold-release', 'off-press', 'off-hold', 'off-hold-release']),
+        exposes.numeric('duration', ea.STATE).withUnit('s')],
```

Comment:
Duration is legacy and has been removed (https://www.zigbee2mqtt.io/devices/324131092621.html#options)

Reply:
What was the reason for that?
Did I missed something because it is send by the devices without having activate legacy-api.

Reply:
> Did I missed something because it is send by the devices without having activate legacy-api.

The legacy api is enabled by default, you have to explicitly disable it as mentioned on https://www.zigbee2mqtt.io/devices/324131092621.html#options

> Is there an alternative, that I can use?

`action_duration`

Reply:
> The legacy api is enabled by default, you have to explicitly disable it as mentioned on

I thought, it's enough to disable legacy-api generally in settings->advance. Thanks. 
What is the diference between disabeling legacy-api in settings and disabeling legacy in devices?

> action_duration

Thanks, I'll try that.

Reply:
The legacy api is the old MQTT api (has nothing to do with this)

Reply:
Thanks, I switched from duration to action_duration.

Reply:
Hi Koenk,

I'm really thankfull for your support. That are mistakes, I do only once a time. But to my apologies, I am really a complete beginner at programming. 
Hopefully now it's all okay. On my local machine, it's running well.

Many, many thanks.